LibAddonMenu-2.0/LibAddonMenu-2.0/controls/description.lua

```diff
@@ -66,6 +68,19 @@ function LAMCreateControl.description(parent, descriptionData, controlName)
     else
         desc:SetAnchor(TOPLEFT)
     end
+    
+    if descriptionData.enableLinks then
+        desc:SetMouseEnabled(true)
+        desc:SetLinkEnabled(true)
+        if type(descriptionData.enableLinks) == "function" then
+            desc.linkClickedHandler = descriptionData.enableLinks
+        else
+            function desc:linkClickedHandler(linkData, linkText, button)
```

Comment:
Please don't use a colon here. When it is written like that, it implies that `self` refers to the `desc` table, but since you pass it to SetHandler as a function reference, it is not actually called on the `desc` table and instead `control` is passed as `self`. Instead just assign an anonymous function to `linkClickedHandler` like you do in the case above and add `control` as the first parameter.